### PR TITLE
Fix write latency in REGIONAL x REGIONAL BY ROW row

### PR DESCRIPTION
**Pull request by stbof**
https://cockroachlabs.atlassian.net/browse/DOC-2714 is addressed in line 48 of choosing-a-multi-region-configuration.md.

Other fixes include:
- Add a few headings for navigation.
- Swap "this" for referenced nouns.
- Add link to relevant blog in See also.

### CODE REVIEW

**Review by ajstorm — CHANGES_REQUESTED** [review]
Looking good.  A few comments below.

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13090#-:-Mx6Vbb4AGI1b6_O07ci:b-7rnzil)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @stbof)

___
*[_includes/v21.2/misc/multiregion-max-offset.md, line 1 at r1](https://reviewable.io/reviews/cockroachdb/docs/13090#-Mx6TmdD6xrO1PDjcq76:-Mx6TmdD6xrO1PDjcq77:b-fze6mz) ([raw file](https://github.com/cockroachdb/docs/blob/4fb8c6e10888ab24cfcb7bb628152534ae101da9/_includes/v21.2/misc/multiregion-max-offset.md#L1)):*
> ```Markdown
> For new clusters using the [multi-region SQL abstractions](multiregion-overview.html), Cockroach Labs recommends lowering the [`--max-offset`](cockroach-start.html#flags-max-offset) setting to `250ms`. This setting is especially helpful for lowering the write latency of [global tables](multiregion-overview.html#global-tables). Changing the setting will require restarting all of the nodes in your cluster at the same time; it cannot be done with a rolling restart.
> ```

Does it make sense to start this last sentence with "For existing clusters, changing this setting...".  For new clusters, if you set this on all nodes as they're first started up, this last sentence shouldn't apply.
___
*[v21.2/choosing-a-multi-region-configuration.md, line 48 at r1](https://reviewable.io/reviews/cockroachdb/docs/13090#-Mx6UGg-3R_HDLB9uU0T:-Mx6UGg-3R_HDLB9uU0U:b-4wmzh4) ([raw file](https://github.com/cockroachdb/docs/blob/4fb8c6e10888ab24cfcb7bb628152534ae101da9/v21.2/choosing-a-multi-region-configuration.md#L48)):*
> ```Markdown
> | `REGIONAL BY TABLE`                                                 | Low-latency for single-region writes and multi-region stale reads.                                                        | Single-region writes are higher latency than for `ZONE`, as at least one additional region must be consulted for each write. Stale multi-region reads are of comparable latency to `ZONE` survival.                  |
> |                                                                     | Single-region apps that can accept region failure.                                                                        | Single-region apps that must survive region failure.                                                                                                                                                                 |
> | `REGIONAL BY ROW`                                                   | Low-latency consistent multi-region reads and writes for rows that are homed in specific regions.                         | Low-latency consistent reads from a row's home region. Low-latency consistent [stale reads](follower-reads.html) from outside the row's home region. Higher-latency writes if writing to a row from its home region. |
> ```

I think this last line should read "Higher-latency writes if writing to a row from _outside_ its home region"

File: _includes/v21.2/misc/multiregion-max-offset.md [hunk 1]
```diff
@@ -1 +1 @@
-For new clusters using the [multi-region SQL abstractions](multiregion-overview.html), we recommend lowering the [`--max-offset`](cockroach-start.html#flags-max-offset) setting to `250ms`.  This is especially helpful for lowering the write latency of [global tables](multiregion-overview.html#global-tables). Note that this will require restarting all of the nodes in your cluster at the same time; it cannot be done with a rolling restart.
+For new clusters using the [multi-region SQL abstractions](multiregion-overview.html), Cockroach Labs recommends lowering the [`--max-offset`](cockroach-start.html#flags-max-offset) setting to `250ms`. This setting is especially helpful for lowering the write latency of [global tables](multiregion-overview.html#global-tables). Changing the setting will require restarting all of the nodes in your cluster at the same time; it cannot be done with a rolling restart.
```
Comment (stbof) [inline _includes/v21.2/misc/multiregion-max-offset.md:1]:
```suggestion
For new clusters using the [multi-region SQL abstractions](multiregion-overview.html), Cockroach Labs recommends lowering the [`--max-offset`](cockroach-start.html#flags-max-offset) setting to `250ms`. This setting is especially helpful for lowering the write latency of [global tables](multiregion-overview.html#global-tables). For existing clusters, changing the setting will require restarting all of the nodes in your cluster at the same time; it cannot be done with a rolling restart.
```

File: v21.2/choosing-a-multi-region-configuration.md [hunk 2]
```diff
@@ -22,29 +24,31 @@ The options for configuring your multi-region cluster include:
...
+|                                                                     | Single-region apps that can accept region failure.                                                                        | Single-region apps that must survive region failure.                                                                                                                                                                 |
+| `REGIONAL BY ROW`                                                   | Low-latency consistent multi-region reads and writes for rows that are homed in specific regions.                         | Low-latency consistent reads from a row's home region. Low-latency consistent [stale reads](follower-reads.html) from outside the row's home region. Higher-latency writes if writing to a row from its home region. |
```
Comment (stbof) [inline v21.2/choosing-a-multi-region-configuration.md:48]:
```suggestion
| `REGIONAL BY ROW`                                                   | Low-latency consistent multi-region reads and writes for rows that are homed in specific regions.                         | Low-latency consistent reads from a row's home region. Low-latency consistent [stale reads](follower-reads.html) from outside the row's home region. Higher-latency writes if writing to a row from outside its home region. |
```

**Comment by stbof** [comment]
> Looking good. A few comments below.
> 
> > Reviewed 2 of 2 files at r1, all commit messages.
> > _[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13090#-:-Mx6Vbb4AGI1b6_O07ci:b-7rnzil)_ status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @stbof)
> 
> _[_includes/v21.2/misc/multiregion-max-offset.md, line 1 at r1](https://reviewable.io/reviews/cockroachdb/docs/13090#-Mx6TmdD6xrO1PDjcq76:-Mx6TmdD6xrO1PDjcq77:b-fze6mz) ([raw file](https://github.com/cockroachdb/docs/blob/4fb8c6e10888ab24cfcb7bb628152534ae101da9/_includes/v21.2/misc/multiregion-max-offset.md#L1)):_
> 
> > ```gfm
> > For new clusters using the [multi-region SQL abstractions](multiregion-overview.html), Cockroach Labs recommends lowering the [`--max-offset`](cockroach-start.html#flags-max-offset) setting to `250ms`. This setting is especially helpful for lowering the write latency of [global tables](multiregion-overview.html#global-tables). Changing the setting will require restarting all of the nodes in your cluster at the same time; it cannot be done with a rolling restart.
> > ```
> 
> Does it make sense to start this last sentence with "For existing clusters, changing this setting...". For new clusters, if you set this on all nodes as they're first started up, this last sentence shouldn't apply.
> 
> _[v21.2/choosing-a-multi-region-configuration.md, line 48 at r1](https://reviewable.io/reviews/cockroachdb/docs/13090#-Mx6UGg-3R_HDLB9uU0T:-Mx6UGg-3R_HDLB9uU0U:b-4wmzh4) ([raw file](https://github.com/cockroachdb/docs/blob/4fb8c6e10888ab24cfcb7bb628152534ae101da9/v21.2/choosing-a-multi-region-configuration.md#L48)):_
> 
> > ```gfm
> > | `REGIONAL BY TABLE`                                                 | Low-latency for single-region writes and multi-region stale reads.                                                        | Single-region writes are higher latency than for `ZONE`, as at least one additional region must be consulted for each write. Stale multi-region reads are of comparable latency to `ZONE` survival.                  |
> > |                                                                     | Single-region apps that can accept region failure.                                                                        | Single-region apps that must survive region failure.                                                                                                                                                                 |
> > | `REGIONAL BY ROW`                                                   | Low-latency consistent multi-region reads and writes for rows that are homed in specific regions.                         | Low-latency consistent reads from a row's home region. Low-latency consistent [stale reads](follower-reads.html) from outside the row's home region. Higher-latency writes if writing to a row from its home region. |
> > ```
> 
> I think this last line should read "Higher-latency writes if writing to a row from _outside_ its home region"

Fixed both.

**Review by ajstorm — APPROVED** [review]
Thanks for making this change.

:lgtm: 

> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/13090#-:-Mx6biMtCLAmRvQRrPA3:b-8x6q0k)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ajstorm and @stbof)

___
*[v21.2/choosing-a-multi-region-configuration.md, line 48 at r1](https://reviewable.io/reviews/cockroachdb/docs/13090#-Mx6UGg-3R_HDLB9uU0T:-Mx6b_TKEIYAc3NujLVR:bb6ywmx) ([raw file](https://github.com/cockroachdb/docs/blob/4fb8c6e10888ab24cfcb7bb628152534ae101da9/v21.2/choosing-a-multi-region-configuration.md#L48)):*
<details><summary><i>Previously, stbof (Stephanie Bodoff) wrote…</i></summary><blockquote>

Fixed both.
</blockquote></details>

LGTM

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**